### PR TITLE
feat: Add support for testimonials - main prompt

### PR DESCRIPTION
**Pull request by Pradumnasaraf**
- Add support for creating testimonials from the CLI (main prompt)

Feel free to review when you are free @Ananya2001-an, thanks!

### CODE REVIEW

File: src/givetestimonial/giveTestimonial.js [hunk 1]
```diff
@@ -1,5 +1,114 @@
...
+    {
+      type: "input",
+      name: "testimonial",
+      message: "What is your testimonial?",
+    },
+  ]);
+  const { testimonial } = answers;
+  if (testimonial === "") {
+    console.log(chalk.bgRed.bold(` Please enter a valid testimonial. `));
+    testimonialdata();
```
Comment (Ananya2001-an) [inline src/givetestimonial/giveTestimonial.js:108]: It should be testimonialdescription();

File: src/givetestimonial/giveTestimonial.js [hunk 2]
```diff
@@ -1,5 +1,114 @@
...
-  console.log("Adding a testimonial");
-};
+const enquirer = require("enquirer");
+const fs = require("fs");
+const chalk = require("chalk");
+const createtestimonialfile = require("./helper/createTestimonialFile");
+const checktestimonial = require("./helper/checkTestimonial");
 
-module.exports = updateJson;
+let testimonialWritter;
```
Comment (Ananya2001-an) [inline src/givetestimonial/giveTestimonial.js:7]: You can fix the typo here for writer

**Review by Ananya2001-an — COMMENTED** [review]
Everything looks good 👍. Just make some small changes that I have marked.

File: src/givetestimonial/giveTestimonial.js (hunk 1 above)
Comment (Pradumnasaraf) [reply src/givetestimonial/giveTestimonial.js:108]: Oh, great catch. Thanks

**Comment by Pradumnasaraf** [comment]
Hey, 

Thanks for the review. I am done with your requested changes

**Comment by Ananya2001-an** [comment]
> Hey, 
> 
> Thanks for the review. I am done with your requested changes

Looks good to merge I guess.

**Comment by Pradumnasaraf** [comment]
Awesome. Let's get it merged.

**Comment by Pradumnasaraf** [comment]
Wanna approve this? You will get a green as well :)

**Comment by Ananya2001-an** [comment]
> Wanna approve this? You will get a green square as well :)

Yeah wait lemme do it😭

**Review by Ananya2001-an — APPROVED** [review]
Awesome work @Pradumnasaraf Couldn't have been any better😂

**Comment by Pradumnasaraf** [comment]
> Awesome work @Pradumnasaraf Couldn't have been any better😂

LOL 🤣 . Don't know if this is a compliment or not. Still, thanks

**Comment by Ananya2001-an** [comment]
> > Awesome work @Pradumnasaraf Couldn't have been any better😂
> 
> LOL 🤣 . Don't know if this is a compliment or not. Still, thanks

No sarcasm intended👍

**Comment by Pradumnasaraf** [comment]
Merged

**Pull request merged**